bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "require-css",
-  "version": "0.1.4",
+  "version": "0.1.8",
```

Comment:
Why do u update to 0.1.8, when latest master does have this already?Looks like u didn't pull latest master or rebased. Have u? The same with `package.json`. Ideally, if u pull/rebase this file should NOT be in changed list. Please check.